core/prisma/scripts/comment-generator.mts

```diff
@@ -23,8 +23,8 @@ import { promises as fs } from "fs";
 
 import type { DMMF, EnvValue, GeneratorOptions } from "@prisma/generator-helper";
 
-import { generatorHandler } from "@prisma/generator-helper";
-import { getDMMF, parseEnvValue } from "@prisma/internals";
+import Helper from "@prisma/generator-helper";
+import Internals from "@prisma/internals";
```

Comment:
similar to the `pg` imports in `core/kysely/db`, prisma has some weird export defined or something where this will fail if you do 

```ts
import { parseEnvValue } from '@prisma/generate-helper';
```